page-factory-core/src/main/java/ru/sbtqa/tag/pagefactory/stepdefs/SetupStepDefs.java

```diff
@@ -15,8 +15,17 @@
 
 public class SetupStepDefs {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Add a private constructor to hide the implicit public one. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1118)